bin/dx-unpack

```diff
@@ -38,7 +38,7 @@ while [[ $# > 0 ]]; do
         output="$2"
         shift 2
     else
-        A="$A $1"
+        A="$A \"$1\""
```

Comment:
As per my investigation, the problem is here with quoting of the arguments. 
Fixed it by quoting it twice. 
ex.
dx-unpack "test 'bundle' \"with\" \"$@#^&%()[]{}\" spaces.tar.gz"
>>> Unpacking test 'bundle' "with" "$@#^&%()[]{}" spaces.tar.gz to /home/abhiram/test-concepts/tmp